lib/zendesk_apps_support/assets/installed.js.erb

```diff
@@ -1,6 +1,28 @@
 (function() {
+  var self = this;
+  var reporter = window.ZendeskReporter ? new window.ZendeskReporter() : {};
+  function wrapped$() {
+    reporter.increment && reporter.increment(
+      'app_framework.app_scope.violated',
+      1,
+      ['origin:' + self.location.hostname]
+    );
+    return self.$.apply(null, arguments);
+  }
+
+  var wrappedWindow = this.$.extend(true, {}, self, { $: wrapped$ });
+  wrappedWindow.window = wrappedWindow;
+  wrappedWindow.top = wrappedWindow;
+
   <% appsjs.each do |appjs| %>
-    <%= appjs %>
+    <% if zaf_naughty_v1_logging %>
+      <%= appjs.gsub(
```

Comment:
Is it possible to (easily) test .erb templates? If so, it'd be nice to see a simple test for this regex

Reply:
👍 I'll add a test case for this.